### ISSUE TITLE
addition of more advance methods

### ISSUE THREAD

**Issue by saeedahmadicp** [saeedahmadicp/numerical_optimization_of_ai#17]
Any collaborator is most welcome to contribute additional advanced methods to this repository.

**Comment by tayyabkhalil-313** [comment]
I am interested in working on this - do you have any algorithms in mind?

**Comment by saeedahmadicp** [comment]
@tayyabkhalil-313 Yes, We want to add advanced techniques to the optimization algorithms such as Genetic Algorithms, Particle Swarm Optimization, Quasi-Newton Methods, etc.
